dstk-web/src/routes/create-model-version/api/createModelVersion.ts

```diff
@@ -0,0 +1,13 @@
+import { gql, useMutation, type TypedDocumentNode } from '@apollo/client';
+
+import { MLModelVersion } from '@/types/MLModelVersion';
+
+const CREATE_MODEL_VERSION: TypedDocumentNode<MLModelVersion> = gql`
+    mutation CreateModelVersion($data: ModelVersionInput) {
+        createModelVersion(data: $data) {
+            numericVersion
```

Comment:
Any use returning the uuid also?

Reply:
I didn't think so. I was going to use the model version in the notification.

Something like:
"Congrats dude! Version X has been successfully created!"

Reply:
Oh, actually, maybe I should. I'm going to put a link to the file upload page on the notification as well, and I should probably use the version ID in the url (you could also use the version number, but I'm using the model ID for the url so this would keep route conventions consistent).

I'm going to push that through and then merge this.